modules/cnf-image-based-upgrade-backup-guide.adoc

```diff
@@ -20,7 +20,7 @@ The following resources must be excluded from the backup:
 
 There are two local storage implementations for {sno}:
 
-Local Storage Operator (LSO):: The {lcao} backs up and restores the required artifacts, including `LocalVolumes` and their associated `StorageClasses`. You must exclude the `persistentVolumes` resource in the application `Backup` CR.
+Local Storage Operator (LSO):: The {lcao} automatically backs up and restores the required artifacts, including `LocalVolumes` and their associated `StorageClasses`. You must exclude the `persistentVolumes` resource in the application `Backup` CR.
```

Comment:
```suggestion
Local Storage Operator (LSO):: The {lcao} automatically backs up and restores the required artifacts, including `LocalVolume` resources and their associated `StorageClass` resources. You must exclude the `PersistentVolumes` resource in the application `Backup` CR.
```
I think "`LocalVolume` resources" and "`StorageClass` resources" should be formatted this way, per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references:
![image](https://github.com/user-attachments/assets/1208fb3d-eb06-4fa5-8c8d-1a651f727894)

Also, I think `PersistentVolumes` should be capitalized, per other references, for example: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reserving-a-persistentvolume and https://docs.openshift.com/container-platform/4.16/storage/persistent_storage/persistent_storage_local/persistent-storage-local.html#local-create-cr-manual_persistent-storage-local

Reply:
Re capitalization:
I'm not sure that in the Backup/Restore CRs you need to capitalize the resource names. Here's where the examples are from: https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/backuprestore-with-oadp.md#application-backup-and-restore-crs

@Missxiaoguo Could you please take a look and confirm whether we need to capitalize in this case?

Reply:
The resources in the Backup/Restore CR could be either capitalized or not. For example,  all of these should work.  
```
  names:
    kind: LocalVolume
    plural: localvolumes
    singular: localvolume
```